client/src/main.rs

```diff
@@ -104,7 +104,7 @@ fn main() {
 						.global(true)
 						.takes_value(true)
 						.value_name("STRING")
-						.default_value("ws://127.0.0.1")
+						.default_value("wss://127.0.0.1")
```

Comment:
Switch to secure web-sockets. It remains to be seen in the integration tests, if the client supports WSS so seamlessly.. (the `ws-rs` client in theory has WSS support as far as I can tell)